cmd/query/app/querysvc/v2/querysvc/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package querysvc
```

Comment:
@yurishkuro is this file path good? currently the adjusters are in `querysvc/v2/adjuster/` - should we also move it under `querysvc/v2/querysvc/adjuster?`

Reply:
V2/adjuster is good path. But now that you mentioned, I think we could move the whole thing out of query/app into query/internal, as part of the overall cleanup I'm pushing for. But we can do it  in a follow up PR.

Reply:
sounds good - will do in a follow-up PR